### PR TITLE
Update flake inputs

### PR DESCRIPTION
**Pull request by SuperSandroBot**
This PR contains the following updates:

| Package | Update | Change |
|---|---|---|
| authentik | digest | `91ff8d9` -> `9d9c0a3` |
| disko | digest | `da52cf4` -> `785c1e0` |
| flake-parts | digest | `506278e` -> `205b12d` |
| home-manager | digest | `a46e702` -> `65912bc` |
| ifstate | digest | `2cb8caf` -> `fe268c4` |
| microvm | digest | `b1fd970` -> `5f0ab79` |
| nix-darwin | digest | `61cee20` -> `55d0781` |
| nixos-apple-silicon | digest | `3eee753` -> `e8c07c3` |
| nixos-hardware | digest | `672ac2a` -> `cceee0a` |
| nixos-modules | digest | `2946b1a` -> `39d26dd` |
| nixpkgs | digest | `5e4fbfb` -> `d0797a0` |
| nixvim | digest | `c674f10` -> `6348336` |
| rust-overlay | digest | `7128722` -> `020701e` |
| search | digest | `fe07070` -> `68e9fad` |
| sops-nix | digest | `e39947d` -> `c6134b6` |
| tsnsrv | digest | `1ee57d5` -> `6ac3562` |

---

### Configuration

📅 **Schedule**: Branch creation - "before 12am on sunday" (UTC), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

👻 **Immortal**: This PR will be recreated if closed unmerged. Get [config help](https://redirect.github.com/renovatebot/renovate/discussions) if that's undesired.

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://redirect.github.com/renovatebot/renovate).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
We need to PR authentik to remove mdDocs

**Comment by MarcelCoding** [comment]
https://github.com/nix-community/authentik-nix/pull/39

**Comment by MarcelCoding** [comment]
https://codeberg.org/kampka/nix-flake-crowdsec/pulls/21

**Comment by SuperSandro2000** [comment]
We probably need to change the lib.attrValues used with nixos-hardware to somehow not trigger the throw. Maybe tryEval? I am not sure.

File: flake.nix [hunk 1]
```diff
@@ -11,7 +11,8 @@
       };
     };
     authentik = {
-      url = "github:nix-community/authentik-nix";
+      # TODO: switch back to nix-community
```
Comment (SuperSandro2000) [inline flake.nix:14]:
```suggestion
      # TODO: switch back to nix-community
      # url = "github:nix-community/authentik-nix";
```

**Comment by SuperSandro2000** [comment]
Let see what now breaks 😂

File: flake.nix [hunk 2]
```diff
@@ -11,7 +11,9 @@
...
```
Comment (SuperSandro2000) [inline flake.nix:14]:
```suggestion
      # TODO: switch back to nix-community
      # https://github.com/nix-community/authentik-nix/pull/39
```

**Pull request merged**